docs/tools-offline-instructions/info/projectModules.json

```diff
@@ -81,6 +87,9 @@
         "^3.0.2",
         "3.0.2"
     ],
+    "sqlite3": [
```

Comment:
Why is this required?

Reply:
There is a script running to collect all dependencies from generated projects. Not sure if I should actively filter out anything. I PM you some details.

Reply:
Thanks for the offline explanation.